stockflux-securities-master/src/components/securities-table/TableRow.js

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import classNames from "classnames";
+import ToolTip from "../tool-tip/ToolTip";
+import {
+  FaPen,
+  FaTrashAlt,
+  FaEye,
+  FaEyeSlash,
+  FaCheck,
+  FaTimes
+} from "react-icons/fa";
+import PropTypes from "prop-types";
+import "./TableRow.css";
+
+const TableRow = ({ item, deleteSecurity, patchSecurity }) => {
+  const handleDelete = () => {
+    deleteSecurity(item.securityId);
+  };
+
+  const handleToggleVisible = () => {
+    patchSecurity(item.securityId, {
+      visible: !item.visible
+    });
+  };
+
+  const handleToggleEnabled = () => {
+    patchSecurity(item.securityId, {
+      enabled: !item.enabled
+    });
+  };
+
+  return (
+    <div className="securities-table-row">
+      <div className="securities-table-cell">{item.exchange}</div>
+      <div className="securities-table-cell">{item.symbol}</div>
+      <div className="securities-table-cell">{item.name}</div>
+      <div className="securities-table-cell">
```

Comment:
You could also do similar mapping here using `Object.keys`. Again - just a suggestion.

Reply:
I don't know if this will work as well as for the headings as `Object.keys` gives all the keys of the securities item but in the object the fields are not in the correct order to display and there are certain fields which do are not displayed such as `securityId`, `updated`, `visible` and `enabled`.

How about `[item.exhange, item.symbol, item.name].map(cellData => <div className="securities-table-cell">{item.exchange}</div>)` ?